pkg/kando/stream_push_test.go

```diff
@@ -0,0 +1,49 @@
+package kando
+
+import (
+	. "gopkg.in/check.v1"
+)
+
+type StreamPushTestSuite struct{}
+
+var _ = Suite(&StreamPushTestSuite{})
+
+func (s *StreamPushTestSuite) TestEnclosePassword(c *C) {
+
+	for _, tc := range []struct {
+		input, output string
+	}{
+		// we dont handle back ticks yet
+		{
+			input:  "this-is3543%$%#$#()*&)~~`-dummy-pass4534",
+			output: "\"this-is3543%$%#$#()*&)~~\\`-dummy-pass4534\"",
```

Comment:
Does this cause a mismatch between the original password and the transformed password? What will happen if a repo is created with an older format of password and after an upgrade, we try to connect to the repo with a transformed password?

Reply:
@PrasadG193, 
the password that would eventually use the password that user provides. We are not actually transforming the password. I still have to test the restore though.

Reply:
Restore works as expected.